src/pages/blogs-overview/BlogsOverviewPage.jsx

```diff
@@ -0,0 +1,65 @@
+import './BlogsOverviewPage.css';
+
+// Eigen hooks
+import useGet from "../../hooks/useGet.jsx";
+
+// Components
+import BlogListItem from "../../components/blog-list-item/BlogListItem.jsx";
+
+function BlogsOverviewPage() {
+  const {data: blogs, getLoading, getError} = useGet('/api/blogs');
```

Comment:
Wellicht is een andere hook-naam beter, hoe vaker ik `useGet` lees, hoe vreemder ik het vind haha